Source/Extensions/Blazorise.QRCode/Enums/EciMode.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Blazorise.QRCode
+{
+    public enum EciMode
```

Comment:
Missing comment, No idea why this does? Some kind of encoding?
Apparently it is set but not used?

Reply:
This is mainly used to describe the Payload parameter and define encoding. I have added comments.